editor/project_manager/project_list.cpp

```diff
@@ -190,9 +193,6 @@ void ProjectListItemControl::set_is_favorite(bool p_favorite) {
 }
 
 void ProjectListItemControl::set_is_missing(bool p_missing) {
-	if (project_is_missing == p_missing) {
```

Comment:
Is this really necessary? What does this fix?

Reply:
With this check every project in list skips the code below. Idk why this check exists, before refactor there was no such check.

Reply:
Just wanted to know if it was part of the solution, as the PR description didn't mention anything!

Reply:
Thanks for clarifying! I'll add more details into pr description.